rkt/run.go

```diff
@@ -110,10 +110,11 @@ func runRun(cmd *cobra.Command, args []string) (exit int) {
 		return 1
 	}
 
-	if globalFlags.Dir == "" {
+	dataDir := getDataDir()
```

Comment:
I wonder if this can be empty at all, but nevermind. This can be solved in other PR I guess.